tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/controller.go

```diff
@@ -102,7 +102,7 @@ func initCache(ctx context.Context, kubeClientSet kubernetes.Interface, params d
 	disabled, err := loadCacheConfig(ctx, kubeClientSet, &params)
 	if err != nil {
 		logger.Errorf("Config map could not be loaded. Error : %w", err)
-		params.LoadMySQLDefaults()
+		// params.LoadMySQLDefaults()
```

Comment:
Why are you commenting out the load default parameters? If we don't need it we can just remove this line and add a proper comment.

Reply:
We are setting defaults in config maps, do you think we should hardcode defaults in the code? So I will get rid of this comment.